scarpe-components/lib/scarpe/components/asset_server.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
```

Comment:
This is top-notch Noah. I would never have thought to construct a server for larger images. Let alone known the proper way to whip one up! Thank you!